### ISSUE TITLE
metalink: glfn for xcache should point to the file

### ISSUE THREAD

**Issue by mlassnig** [rucio/rucio#1483]
Motivation
----------

Currently it points to the archive, should point to the file inside the zip instead.

**Comment by mlassnig** [comment]
suggestion from xcache devs (@abh3, @wyang007):

```
<glfn name="/atlas/rucio/mock:another.one.zip?xrdcl.unzip=zippedfile-0-5b7217bc734b41ed9547162d6693e5d2"></glfn>
<url location="APERTURE_MOCK_LOSSXLHIQX" domain="zip" priority="1"
    client_extract="false">root://root.aperture.com:1409//test/chamber/mock/d7/b0/another.one.zip?xrdcl.unzip=zippedfile-0-5b7217bc734b41ed9547162d6693e5d2</url>
```

**Comment by abh3** [comment]
Hi Mario,

Indeed, that was what I recommended today for Wei', conundrum. In general 
we want to know the base file not the archive within it as we will be 
cachin the whole zip file. That said, there are a lot of devious issues 
involved here, especially when dealing with intermediate servers along the 
way. So, I can't say we have addressed all the issues. However, as a first 
take, what you say is the most reasonable option.

Andy
